src/gh_issue_487.rs

```diff
@@ -0,0 +1,49 @@
+
+use pnet_packet::Packet;
+use pnet_packet::ip::IpNextHeaderProtocol;
+use pnet_packet::ipv6;
+
+
+#[test]
+fn main() {
```

Comment:
Does this get picked up when running `cargo test`?

Reply:
yes

Reply:
AFAIK